AgentDiscoveries-Backend/src/main/java/org/softwire/training/api/models/TokenResponseApiModel.java

```diff
@@ -6,12 +6,14 @@
     private String expires;
     private int userId;
     private boolean isAdmin;
+    private int agentId;
 
-    public TokenResponseApiModel(String token, String expires, int userId, boolean isAdmin) {
+    public TokenResponseApiModel(String token, String expires, int userId, boolean isAdmin, int agentId) {
         this.token = token;
         this.userId = userId;
         this.expires = expires;
-        this.isAdmin=isAdmin;
+        this.isAdmin = isAdmin;
+        this.agentId = agentId;
```

Comment:
I feel like we should add a getter to this field so it's consistent with the others?

Reply:
added